apidoc/Titanium/Android/HttpResponseCache.yml

```diff
@@ -0,0 +1,149 @@
+        in the Android API Reference.
+    returns:
+        type: Number
+
+  - name: getNetworkCount
+    summary: Returns the number of HTTP requests that required the network to either supply a response or validate a locally cached response.
+    description: |
+        See also:
+        [getNetworkCount](https://developer.android.com/reference/android/net/http/HttpResponseCache.html#getNetworkCount())
+        in the Android API Reference.
+    returns:
+        type: Number
+
+  - name: getRequestCount
+    summary: Returns the total number of HTTP requests that were made.
+    description: |
+        See also:
+        [getRequestCount](https://developer.android.com/reference/android/net/http/HttpResponseCache.html#getRequestCount())
+        in the Android API Reference.
+    returns:
+        type: Number
+
+  - name: size
+    summary: Returns the number of bytes currently being used to store the values in this cache.
+    description: |
+        See also:
+        [size](https://developer.android.com/reference/android/net/http/HttpResponseCache.html#size())
+        in the Android API Reference.
+    returns:
+        type: Number
+
+  - name: getHttpCachePath
+    summary: Gets relative path for cache.
+    description: |
+        Default value `http`.
+    returns:
+        type: String
+
+  - name: getHttpCacheSize
+    summary: Returns the maximum number of bytes that this cache should use to store its data.
+    description: |
+        Default value is 25MB. This could be redefined by
+        [setHttpCacheSize](Titanium.Android.HttpResponseCache.setHttpCacheSize) or by setting
+        `ti.android.cache.size.max` property in `tiapp.xml`
+        See also:
+        [maxSize](https://developer.android.com/reference/android/net/http/HttpResponseCache.html#maxSize())
+        in the Android API Reference.
+    returns:
+        type: Number
+
+  - name: setHttpCachePath
+    summary: Sets relative path for cache.
+    description: |
+        Override default cache path.
+    parameters:
+      - name: path
+        summary: Relative path for cache.
+        type: String
+
+  - name: SetHttpCacheSize
+    summary: Sets the maximum number of bytes that this cache should use to store its data.
+    description: |
+        Should be set before calling [install](Titanium.Android.HttpResponseCache.install) method
+    parameters:
+      - name: size
+        summary: Maximum cache size in bytes.
+        type: Number
+
+properties:
+
+  - name: httpCacheSize
+    summary: Maximum  cache size in bytes.
+    description: |
+        See also:
+        [maxSize](https://developer.android.com/reference/android/net/http/HttpResponseCache.html#maxSize())
+        in the Android API Reference.
+    type: Number
+
+  - name: httpCachePath
+    summary: Relative path for cache
+    description: |
+        HTTP response cache will be stored on external storage if it exists on moment
```

Comment:
Same as above, maybe more more explicit that this property is consulted at time `#install()` is called, but changing later won't do anything. (In fact we may even want to spit out some warning logs in the impl if they get called after `#install()`).

Reply:
Add this note:
>`To apply changes of this property to existing HttpResponseCache instance, you should call         [install](Titanium.Android.HttpResponseCache.install) method.`